assets/templates/custom_fields/CERNFields.js

```diff
@@ -0,0 +1,76 @@
+// This file is part of Invenio-RDM-Records
+// Copyright (C) 2020-2023 CERN.
+// Copyright (C) 2020-2022 Northwestern University.
+//
+// Invenio-RDM-Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+
+import { FieldLabel, Input } from "react-invenio-forms";
+import { Divider, Grid, GridColumn } from "semantic-ui-react";
+
+import PropTypes from "prop-types";
+import { AccordionField } from "react-invenio-forms";
+
+export class CERNFields extends Component {
+  render() {
+    const { key, children, includesPaths, label, active } = this.props;
+    return (
+      <AccordionField
+        key={key}
+        includesPaths={includesPaths}
+        label={label}
+        active={active}
+      >
+        <FieldLabel htmlFor="CERNFields" icon="building outline" label="Unit" />
+        <Divider fitted />
+        <Grid padded>
+          <Grid.Column computer={16}>{children[0]}</Grid.Column>
```

Comment:
nit: it would be more readable just if you export these to `const [x, y, z] = children`